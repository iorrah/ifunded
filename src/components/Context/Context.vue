<template>
  <div class="context container-fluid">
    <h1 class="context__title">Platform users</h1>

    <ul class="context__menu">
      <li class="context__menu-item">
        <a href="#" class="context__menu-link">All users</a>
        <i class="context__menu-indicator--active"></i>
      </li>

      <li class="context__menu-item">
        <a href="#" class="context__menu-link">Created by me</a>
      </li>

      <li class="context__menu-item">
        <a href="#" class="context__menu-link">Deleted</a>
      </li>
    </ul>
  </div>
</template>

<script>
export default {
  name: 'Context',
};
</script>
