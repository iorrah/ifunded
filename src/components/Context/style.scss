.context {
  display: block;
  width: 100%;
  padding-top: 1.5rem;
  padding-bottom: 1rem;
  background: $turquoise;
  box-shadow: 0 4px 4px #0000000d;
  color: $white;
}

.context__title {
  font-size: 1.7rem;
  font-weight: 500;
  margin: 0 0 1.6rem;
}

.context__menu {
  display: inline-block;
  position: relative;
  padding: 0;
  margin: 0;
}

.context__menu-item {
  list-style: none;
  display: inline-block;
  margin: 0 1.3rem 0;
}

.context__menu-item:first-child {
  margin-left: 0;
}

.context__menu-link {
  color: $white;
  padding: 1.5rem 0;
  font-size: 0.9rem;
  font-weight: 500;
}

.context__menu-link--active {
  color: $white
}

.context__menu-indicator--active {
  background: $white;
  display: block;
  height: 0.2rem;
  width: 100%;
  position: relative;
  bottom: -1rem;
  border-radius: 1rem 1rem 0 0;
}