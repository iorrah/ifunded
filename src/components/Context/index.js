// eslint-disable-next-line import/newline-after-import
import Context from './Context.vue';
export default Context;
