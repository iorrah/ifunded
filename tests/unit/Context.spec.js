import { mount } from '@vue/test-utils';
import Context from '../../src/components/Context/index';

describe('Context', () => {
  test('renders correctly', () => {
    const wrapper = mount(Context);
    expect(wrapper.element).toMatchSnapshot();
  });
});
