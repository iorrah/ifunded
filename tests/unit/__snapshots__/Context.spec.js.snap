// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Context renders correctly 1`] = `
<div
  class="context container-fluid"
>
  <h1
    class="context__title"
  >
    Platform users
  </h1>
   
  <ul
    class="context__menu"
  >
    <li
      class="context__menu-item"
    >
      <a
        class="context__menu-link"
        href="#"
      >
        All users
      </a>
       
      <i
        class="context__menu-indicator--active"
      />
    </li>
     
    <li
      class="context__menu-item"
    >
      <a
        class="context__menu-link"
        href="#"
      >
        Created by me
      </a>
    </li>
     
    <li
      class="context__menu-item"
    >
      <a
        class="context__menu-link"
        href="#"
      >
        Deleted
      </a>
    </li>
  </ul>
</div>
`;
